modules/hcp-support-matrix.adoc

```diff
@@ -0,0 +1,144 @@
+// Module included in the following assemblies:
+//
+// * hosted-control-planes/hcp-prepare/hcp-requirements.adoc
+
+
+:_mod-docs-content-type: CONCEPT
+[id="hcp-support-matrix"_{context}"]
+= Support matrix for {hcp}
+
+Because {mce} includes the HyperShift Operator, releases of {hcp} align with releases of {mce-short}. For more information, see "OpenShift Operator Life Cycles".
```

Comment:
Should "OpenShift Operator Life Cycles" be a hyperlink?

Reply:
We include a link to it in the Additional resources section

Reply:
Can we say we do somehow, as is I wouldn't know.